Support for all different accent colors, Minecraft versions from 1.6.1 to 1.19.3 and lot more.

JustLetterV opened the pull request:
### Fixes:

- Fixed wrong pathname assets/minecraft/items -> assets/minecraft/item, this has been like this since the 1.14 version.
	- This fixes colors of empty armor slots for 1.14+.
- Fixed a lot of small issues
	- Brewing stand progress bar one pixel offset.
	- Few pixels in beacon inventory.
	- Anvil inventory bottom corners not being round.
	- One pixel offset in looms UI and another one pixel offset in banner pattern slot.
	- One pixel offset in anvil UI.
	- One pixel offset in stonecutter UI.
	- One pixel offset in grindstone output slot.
	- Few pixels in advancements UI.
	- One pixel offset in game mode changer UI.
	
- Fixed creative inventory tab names in the language file for 1.19.3+

### Changes:

- Changed current hand slot texture to use an accent color.
- Made changes to all advancement menu textures.
- Changes to anvil UI
	- Item renaming box now only shows up when there is something to rename in anvil.
	- Made different parts of anvil UI connected together for better text visibility when interacting with items in anvil.
- Changed horse/donkey/Llama inventory buttons and inventory look round like every other button/inventory.
- Changed the color of recipe book outline in inventory. Before it was same color as the background so it didn't show up.
- Changes to following textures:
	- Loom
	- Stonecutter
	- Grindstone
	- Enchant table
	- Gamemode switcher
	- Inventory
- Changed all inventory textures to have different sections separated.
- Changed main menu titles to be the same in every flavor for now.

### Additions:

- Python script that automatically creates a resource pack for each flavor and accent color.
	- Support for all accent colors.
	- Creates 56 different versions of the resource pack from one template resource pack.
	- Easier to modify. Only 1 file needs to be changed and it can be updated for each flavor and accent color with this script.
	- Support for all languages Minecraft supports.
		- Folder "lang" to store everything related to language files.
		- Script to import all language files from Minecraft object folders.
		- Language files are per game version, new ones are needed when adding support for new versions.
	- Support for language files of mods.
	
- Added template resource packs for every pack format. All versions since 1.6.1 are now supported :)
	- Folder "template" to store all the template resource packs.
	- Each version has its own template resource pack.
	- Template pack uses colors of flavor mocha for everything other than accent color. Accent color in the template is '#FF0000' (Red).
	- Added template language file to assets/minecraft/lang. (this is used by the script)
		- Added a lot of new entries to language files.
	
	- Each template has a "template_settings.json" file that stores the template version and names of files that the script should ignore.
	
- Added missing textures:
	- Thin-layout version of the effect list texture to the inventory.
	- Freezing hearts to icons.png.
	- Chat tags textures.
	- Streaming indicators.
	- Two missing buttons for fabric mod [Mod Menu](https://modrinth.com/mod/modmenu).
	- Second hand texture when the main hand is left hand.

- Added support for mod [Organizable Play Screens](https://modrinth.com/mod/organizable-play-screens)
- Added language file for mod [Show Me Your Skin!](https://modrinth.com/mod/show-me-your-skin)

JustLetterV committed:
Removed old resourcepack folders

JustLetterV committed:
Few tiny changes or something

- Added support for all MC versions from 1.6.1 to 1.19.3
- Support for all flavors and accent colors :)
- Script to generate resource pack for all flavors and accent colors from one template resource pack.
- Support for all languages Minecraft supports.

JustLetterV committed:
Whoops, didn't mean to add the 56 ready versions of the resource pack for 1.19.3

JustLetterV committed:
Github doesn't like empty folders

- Fixed issue with "temp" folder missing when using script

JustLetterV committed:
Removed unused language files

JustLetterV committed:
Added missing second hand texture

JustLetterV committed:
Added frozen hearts for 1.19.3

JustLetterV committed:
Added support for transparent textures

- Transparent colors can now be used. When changing the colors of pixels in images, new color uses the alpha value of old color in the pixel.
- Changed selector texture of mod ShowMeYourSkin to use transparent texture.

JustLetterV committed:
Support for mod Iris

- Added support for [Iris](https://modrinth.com/mod/iris). Except this doesn't work with current version of Iris, but looks like [1](https://discord.com/channels/774352792659820594/1076148570908016670/1076181286214963310) [2](https://github.com/IrisShaders/Iris/commit/50978a1d9fd268157f24dcedb6dc0b19b29c1cbd) they might add support for changing textures with resourcepacks.

JustLetterV committed:
Support for 1.19.4

- Added support for 1.19.4
- Changed disabled buttons textures for 1.19.4
- Fixed small issue with script used to import language files
- Renamed template folders from "MC <minecraft version>" to just "<minecraft version>".
- Added .gitignore

JustLetterV committed:
This time actually added the .gitignore

JustLetterV committed:
Fixed grindstone texture for 1.19.3 and 1.19.4

JustLetterV committed:
Small change for button colors in beacon inventory

JustLetterV committed:
Updated textures of disabled buttons

Updated textures of disabled buttons for all versions below 1.19.4

JustLetterV committed:
Update .gitignore

JustLetterV committed:
Added support for fabric-item-group-api

JustLetterV committed:
Added more realms textures

JustLetterV committed:
Support for more mods and small changes

- Support for mod Auth Me.
- Support for mod No Chat Reports.
- Added better error messages into script for errors with template language files.
- Updated textures and language files for mod Show Me Your Skin.

Ringlings commented:
> If anyone wants to try the pack with these new changes and give feedback. These are for Minecraft 1.19.3

Looks good, only issue we're having with it is the lack of theming on some modded UI (namely EMI impacts us the most) but once this gets merged further modded edits will be super simple!

JustLetterV committed:
Support for even more mods

- Added missing texture for ModMenu 1.19.4
- Added support for EMI and some of it's addons (EMI Trades, EMI Loot & Emiffect)
- Added support for Cloth Config API
- Added support for Language Reload

JustLetterV committed:
Fixed small issue with texture of Language Reload
